simplemind/integrations/openai.py

```
@@ -1,68 +1,74 @@
-        else:
-            return AIResponse(
-                response=completion,
-                text=completion.choices[0].message.content,
-            )
+        try:
+            completion = self.client.chat.completions.create(**params)
+            return completion
+        except Exception as e:
+            # Enhanced error handling (optional)
+            logger.error(f"OpenAI API Error: {e}")
+            raise RuntimeError(f"Failed to generate response: {e}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)